bin/spartan-env

```diff
@@ -37,7 +44,10 @@ else
     IP=$(ip r g 192.88.99.0 | grep -Po 'src \K[\d.]+')
 fi
 NAME=${APP_NAME}@${IP}
-RELNAME="`dirname \"$0\"`"/${APP_NAME}
+SCRIPT=$(readlink -f $0 || true)
```

Comment:
I've just copied some code from [relx](https://github.com/erlware/relx/blob/master/priv/templates/extended_bin#L31). I hope they have a reason to do that in this way.